Deepgram/Models/Agent/v2/WebSocket/HistoryFunctionCalls.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright 2024 Deepgram .NET SDK contributors. All Rights Reserved.
+// Use of this source code is governed by a MIT license that can be found in the LICENSE file.
+// SPDX-License-Identifier: MIT
+
+using System.Text.Json;
```

Comment:
@lukeocodes  I debated if this should be 2 files or one, based on existing patterns it seems like maybe 2 files one for function call and another for function calls is the way to go, but this approach did work.  So let me know what you think.